website/server/controllers/api-v3/auth.js

```diff
@@ -75,6 +75,10 @@ api.registerLocal = {
       email: {
         notEmpty: {errorMessage: res.t('missingEmail')},
         isEmail: {errorMessage: res.t('notAnEmail')},
+        notEmailDomains: {
```

Comment:
Take a look at how it's done here: https://github.com/HabitRPG/habitrpg/pull/7655/files#diff-1f63728ba8da6291c25b8acf11aab121R32